x-pack/plugins/lists/server/index.ts

```diff
@@ -12,7 +12,10 @@ import { ListPlugin } from './plugin';
 
 // exporting these since its required at top level in siem plugin
 export { ListClient } from './services/lists/list_client';
-export { CreateExceptionListItemOptions } from './services/exception_lists/exception_list_client_types';
+export {
```

Comment:
@spong , @FrankHassanabad - if you get a chance, can you 👍 this here?
I exposed this additional type for use in Trusted Apps service

Reply:
If they don't get to it, this seems completely reasonable. it's :+1: